libraries/UHS_host/UHS_MIDI/examples/USB_HOST_SHIELD/USB_MIDI_dump/USB_MIDI_dump.ino

```diff
@@ -39,30 +40,28 @@
 
 #include <UHS_host.h>
 
-MAX3421E_HOST UsbHost;
-UHS_MIDI Midi(&UsbHost);
+MAX3421E_HOST *UsbHost;
+UHS_MIDI *Midi;
 bool connected;
-char buf[20];
-
 
 void setup() {
-  // USB data switcher, PC -> device. (test jig, this can be ignored for regular use)
-  pinMode(5, OUTPUT);
-  digitalWrite(5, HIGH);
```

Comment:
Please leave the test-jig code in the examples, as it makes testing easier.
It is the same test jig that I sent you, and the goal of the jig is to allow devices under test to remain plugged in, so the usb port on the board is not worn prematurely.